### ISSUE TITLE
Update dependencies

### ISSUE THREAD

**Issue by tayvs** [akka/akka-management#635]
Update following dependencies to last one

- **Akka** 2.5.23 -> 2.5.27
- **Akka-http** 10.1.10 -> 10.1.11
- **Scala 2.13** 2.13.0 -> 2.13.1

Mb other dependencies need update too

**Comment by ihostage** [comment]
It's work for @scala-steward 😄

**Comment by patriknw** [comment]
Intentionally we don't always update to latest versions to be able to support of wider range of version combinations. You can always update to the latest version in your own build by overriding the dependencies.

**Comment by Daxten** [comment]
@patriknw play is on akka-http 10.1.11 since 2.8, and that is incompatible with this version. imo it would make sense to support play out of the box.

**Comment by raboof** [comment]
@Daxten I'm not sure what you mean - since akka-http promises binary compatibility between releases, it should be fine to use akka-management (indeed built against akka-http 10.1.10) with later versions of akka-http (such as 10.1.11, or even 10.2.0 when it's out later).

Did you run into any problems that were caused by the different akka-http versions?

**Comment by Daxten** [comment]
Hey, yes but maybe there is a workaround I don't know about:

```
[error] java.lang.IllegalStateException: Detected possible incompatible versions on the classpath. Please note that a given Akka HTTP version MUST be the same across all modules of Akka HTTP that you are using, e.g. if you use [10.1.11] all other modules that are released together MUST be of the same version. Make sure you're using a compatible set of libraries. Possibly conflicting versions [10.1.10, 10.1.11] in libraries [akka-http-spray-json:10.1.10, akka-parsing:10.1.11, akka-http:10.1.11, akka-http-core:10.1.11]
```

fixed it for now with my own published version with akka-http 10.1.11

**Comment by raboof** [comment]
The fix for that problem is to depend on akka-http-spray-json 10.1.11 instead of 10.1.10
